lib/Cargo.toml

```diff
@@ -0,0 +1,8 @@
+[package]
+name = "monitoring-lib"
```

Comment:
Consider giving crate in `lib/` a better name, e.g. `bridgehead-monitoring-lib`.